mmdeploy/backend/tensorrt/utils.py

```diff
@@ -10,38 +10,68 @@
 from .init_plugins import load_tensorrt_plugin
 
 
-def create_trt_engine(onnx_model: Union[str, onnx.ModelProto],
-                      input_shapes: Dict[str, Sequence[int]],
-                      log_level: trt.Logger.Severity = trt.Logger.ERROR,
-                      fp16_mode: bool = False,
-                      int8_mode: bool = False,
-                      int8_param: dict = None,
-                      max_workspace_size: int = 0,
-                      device_id: int = 0,
-                      **kwargs) -> trt.ICudaEngine:
+def save(engine: trt.ICudaEngine, path: str) -> None:
+    """Serialize TensorRT engine to disk.
+
+    Args:
+        engine (tensorrt.ICudaEngine): TensorRT engine to be serialized.
+        path (str): The absolute disk path to write the engine.
+    """
+    with open(path, mode='wb') as f:
+        f.write(bytearray(engine.serialize()))
+
+
+def load(path: str) -> trt.ICudaEngine:
+    """Deserialize TensorRT engine from disk.
+
+    Args:
+        path (str): The disk path to read the engine.
+
+    Returns:
+        tensorrt.ICudaEngine: The TensorRT engine loaded from disk.
+    """
+    load_tensorrt_plugin()
+    with trt.Logger() as logger, trt.Runtime(logger) as runtime:
+        with open(path, mode='rb') as f:
+            engine_bytes = f.read()
+        engine = runtime.deserialize_cuda_engine(engine_bytes)
+        return engine
+
+
+def from_onnx(onnx_model: Union[str, onnx.ModelProto],
```

Comment:
`from_onnx` sounds like an attribute function of some class. How about onnx2tensorrt_impl or keeping the original name?

Reply:
`torch.from_numpy` is used to transform `ndarray` to `torch.Tensor`. This name follows the same rule.